### ISSUE TITLE
PPA doesn't work on Linux Mint

### ISSUE THREAD

**Issue by m3thm4th** [libredeb/lightpad#33]
PPA can't be added on Linux Mint

**Comment by libredeb** [comment]
Hi @m3thm4th I'm taking the resolution of issues.

Could you (if you remember) tell me the linux mint version? I guess it would be the 21.2 based on ubuntu 22.04 right?

The PPA repository is out of date, I will try to add support for newer versions.

**Comment by libredeb** [comment]
@m3thm4th yes, it would be a dream come true to have LightPad in debian repositories. But it is a long and complex process (which I will try anyway, but it will require patience).

Right now is already available the latest release for ubuntu 22.04 and derived 😊 enjoy it.
